models/stg_mailchimp__segment_members.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(enabled=var('using_automations', True)) }}
```

Comment:
This isn't an `automation` type model. We should probably not include this functionality here.